assets/Yves/default/app/components/customer-reorder-widget/index.js

```diff
@@ -1,26 +0,0 @@
-/**
- * This file is part of the Spryker Suite.
- * For full license information, please view the LICENSE file that was distributed with this source code.
- */
-
-'use strict';
-
-module.exports = {
```

Comment:
When you are dropping it out, you want to drop the feature or hide an error for now?